src/reports/review-reports.md

```diff
@@ -2,21 +2,26 @@
 title: Review Reports
 ---
 
-The Review Reports provide information about product reviews by customer and by product.
+If using the built-in Magento Reviews, Review Reports provide information about product reviews by customer and by product.
 
-* By Customers
-* By Products
+If [Yotpo Reviews]({% link marketing/yotpo-reviews-intro.md %}) is enabled, the built-in Magento Reviews function and reporting (By Customers and By Products) is disabled. [Yotpo Reporting]({% link marketing/yotpo-reporting.md %}) is accessed in your Yotpo account.
 
-## By Customers
+## Magento Reviews
+
+**By Customer**
 
 ![]({% link images/images/reports-marketing-customer-reviews.png %}){: .zoom}
-*Review Report By Customers*
+_Review Report By Customers_
 
 The Customer Reviews Report lists all customer who have submitted product reviews. The report includes the number of reviews submitted by each customer, and a link to the list of reviews.
 
-## By Products
+**By Products**
 
 ![]({% link images/images/reports-marketing-product-reviews.png %}){: .zoom}
-*Review Report by Product*
+_Review Report by Product_
 
 The Product Reviews Report lists all products that have been reviewed by customers. The report includes the number of reviews and average rating, the product was last reviewed, and a link to the list of reviews for each product.
+
+## Yotpo Reviews
+
+[Yotpo](https://www.yotpo.com/) is a user-generated content marketing platform enabling merchants to gather, curate, manage, and respond to all kinds of user content from just a single platform. For more Yotpo information, see [Yotpo Reviews]({% link marketing/yotpo-reviews-intro.md %}) and [Yotpo Reporting]({% link marketing/yotpo-reporting.md %}).
```

Comment:
```suggestion
[Yotpo](https://www.yotpo.com/) is a user-generated content marketing platform that enables merchants to gather, curate, manage, and respond to all kinds of user content from just a single platform. For more Yotpo information, see [Yotpo Reviews]({% link marketing/yotpo-reviews-intro.md %}) and [Yotpo Reporting]({% link marketing/yotpo-reporting.md %}).
```

Reply:
made change directly in file. unable to commit from suggestion